.github/workflows/scheduled-dependency-update.yml

```diff
@@ -0,0 +1,33 @@
+name: Scheduled Dependency Update
+
+on:
+  # run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+  # every monday at 02:00
+  schedule:
+    - cron: '0 2 * * 1'
+
+jobs:
+  dependency-update:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+
+      - name: Upgrade dependencies
+        run: yarn upgrade --latest
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
+        with:
+          token: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+          branch: chore-dependency-update
+          delete-branch: true
+          title: "chore: dependency update"
+          commit-message: "chore: dependency update"
+          body: |
+            This PR was automatically created.
+            - all dependencies are updated
+            - this PR will be created every monday at 2am and force pushed
```

Comment:
force pushed in case the branch gets outdated in the meantime ?

Reply:
Just to minimize the error rate, for example if someone did forget to merge the last PR or something changed in the `package.json` in the meantime.